test/controllers/HomeControllerSpec.scala

```diff
@@ -339,5 +328,108 @@ class HomeControllerSpec extends BaseSpec with CurrentTaxYear {
 
       verify(controller.messageFrontendService, times(1)).getUnreadMessageCount(any())
     }
+
+  }
+
+  "Calling serviceCallResponses" should {
+
+    "return TaxComponentsDisabled status where there is not a PertaxContext and Nino" in new LocalSetup {
```

Comment:
typo here for 

> not a PertaxContext

Also serviceCallResponses does not use a PertaxContext so this can probably be deleted

Reply:
Amended